config/locales/en/views.yml

```diff
@@ -338,3 +338,7 @@ en:
       title: You cancelled this visit
     withdrawn:
       title: You cancelled this visit request
+  staff_info:
+    not_available:
+      staff_info_service_unavailable: >-
+        Staff info service is temporarily unavailable. Please check back later.
```

Comment:
@tatyree this should be in `prison_views.yml` – so that it doesn't get requested as a translation.